Game/src/StaticRect.java

```diff
@@ -0,0 +1,91 @@
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.shape.Rectangle;
+
+import java.util.ArrayList;
+
+
+public class StaticRect extends StaticObject{
+
+    double x;
+    double y;
+    double width;
+    double height;
+    Image sprite;
+
+    ImageView spriteView = new ImageView();
+
+    /**
+     * constructor
+     * @param x x coordinate of rectangle
+     * @param y y coordinate of rectangle
+     * @param width width of rectangle
+     * @param height height of rectangle
+     * @param sprite sprite of rectangle
+     */
+    public StaticRect(double x, double y, double width, double height, Image sprite){
+
+        this.x = x;
+        this.y = y;
+        this.width = width;
+        this.height = height;
+        this.sprite = sprite;
+
+        spriteView.setImage(sprite);
+        spriteView.setX(x);
+        spriteView.setY(y);
+        spriteView.setFitWidth(width);
+        spriteView.setFitHeight(height);
+
+        MainApplication.addToRoot(spriteView);
+
+    }
+
+    /**
+     *
+     *
+     * @param player the player
+     * @return returns colliding on left,right,top and bottom
+     */
+    @Override
+    public ArrayList<CollisionType> checkCollision(Player player) {
+        ArrayList<CollisionType> collisions = new ArrayList<>();
+
+        boolean collided = false;
+
+        if(player.getX() + player.getFitWidth() >= spriteView.getX() &&
```

Comment:
This if statement is quite long. consider adding a comment explaining what it is doing